.htmlhintrc

```diff
@@ -1,9 +1,10 @@
 {
+  "$schema": "https://json.schemastore.org/htmlhint.json",
   // Question files don't need to have a doctype declaration
```

Comment:
I get a warning that `.htmlhintrc` shouldn't contain comments, should I remove this now that there is intellisense?

Reply:
Removed in [b035fac](https://github.com/PrairieLearn/PrairieLearn/pull/11343/commits/b035facc79f5d673e99a3e7bc08e3e0c13e1fefa).

Reply:
Given that this file has been working fine for years, it'd seem to me that the warning is incorrect.